android/jni/tun2socks/tun2socks.c

```diff
@@ -226,7 +228,8 @@ static void init_arguments (const char* program_name);
 // ==== PSIPHON ====
 
 //==== UPROXY ====
-BAddr dns_server_address;  // IP address of local DNS resolver
```

Comment:
We forked tun2socks from Psiphon, there are in the source for purely historic reasons.
Indeed it would be useful to have a single repo with all the changes, we are scoping the amount of work that would be required to achieve something like this. 

I am inclined to keep this initiative separate from this repo and instead track it internally.